### PR TITLE
enable smoke test for es6

### PR DESCRIPTION
**Pull request by jcantrill**
This PR makes a first pass to re-enable a smoke test to do basic es6 testing

### CODE REVIEW

**Comment by ewolinetz** [comment]
what about modifying and using `hack/testing/check-EFK-running.sh` ?

File: hack/deploy-logging.sh [hunk 1]
```diff
@@ -235,24 +239,50 @@ update_images_in_clo_yaml() {
...
+    echo INFO: Expecting $expectedcollectors collector to start
     # we expect $nodeCount elasticsearch pods
+    es_ready="false"
+    fluent_ready="false"
+    kibana_ready="false"
     wait_func() {
-        local actuales=$( oc get pods -l component=elasticsearch 2> /dev/null | grep -c 'elasticsearch.* 2/2 .*Running' )
+        result=0
+        local actuales=$( oc -n ${LOGGING_NS} get pods -l component=elasticsearch 2> /dev/null | grep -c 'elasticsearch.* 2/2 .*Running' )
         if [ $expectedes -ne ${actuales:-0} ] ; then
```
Comment (periklis) [inline hack/deploy-logging.sh]: `$expectedes` seems unset before usage in this condition.

Comment (jcantrill) [reply hack/deploy-logging.sh]: This value is set leading up to the call to this function.  It's not obvious.  I believe this is something to be revisited as we refactor.

**Pull request merged**